.github/workflows/build-image.yml

```diff
@@ -23,7 +23,39 @@ env:
   REGISTRY: ghcr.io
   IMAGE_NAME: llm-d
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
I didn't opt for this because sometimes I push another change after just to update guides or kustomize manifests and I don't need the rebuild. This strategy will mean forcing a new build to complete, but will cut down on the number of jobs we have - not sure where we stand on that tradeoff

Reply:
Sounds great